tests/ui/test_validation_ui.py

```diff
@@ -102,3 +102,17 @@ def test_odf_cephblockpool_compression_status(self, setup_ui_class):
             f"'Compression status' from ocs-storagecluster-cephblockpool = {compression_statuses[1]}\n"
             f"Expected: {compression_status_expected}"
         )
+
+    @ui
+    @pytest.mark.bugzilla("1994584")
+    def test_ocs_operator_is_not_present(self, setup_ui_class):
+        """
+        Validate odf operator is present in the installed operator tab in ui.
+
+         Args:
+            setup_ui_class: login function on conftest file
+
+        """
+
+        validation_ui_obj = ValidationUI(setup_ui_class)
```

Comment:
I think we don't need to pass `setup_ui_class` fixture as per the recent changes made by Daniel in the base_ui.py file. Pls check